src/data/pnat/factors.lean

```diff
@@ -210,7 +210,7 @@ theorem factor_multiset_prod (v : prime_multiset) :
 begin
   apply prime_multiset.coe_nat_injective,
   rw [v.prod.coe_nat_factor_multiset, prime_multiset.coe_prod],
-  rcases v with l,
+  rcases v with ⟨l⟩,
```

Comment:
I see a handful of changes like this, could you explain?

Reply:
The new `rcases` is not compatible with the old one specifically regarding the pattern `rcases e with a`, which was previously the same as `cases e with a` and is now the same as `have a := e`. To get the `cases` behavior again, you now have to write `rcases e with ⟨a⟩`. (This is done to make the grammar more regular, and in preparation for a possible move of `obtain` to core.)

Reply:
Could you write this above the line in the first message of the PR so it appears in the commit message? This is an important note to have in the "changelog."